examples/README.md

```diff
@@ -0,0 +1,109 @@
+The csp-billing-adapter requires a configuration file, (defaults to `~/.config/csp_billing_adapter/config.yaml`)
+in order to run successfully. The configuration files in in the `examples` directory 
+contain examples for the different product configurations. 
+
+This README provides a listing of all of the fields and descriptions.
+Fields that are optional are labeled accordingly.
+
+```
+version:
+    The version of the adapter implementation.
+
+billing_interval:
+    Sets the billing interval time period.
+
+api: (OPTIONAL)
+    Provides the path to the application information the csp-billing-adapter reports to the CSP API.
+    This is shared with the application the csp-billng-adapter is reporting about. The application may provide this as an endpoint directly, i.e. http://localhost/$API works.
+
+product_code:
+    TBD
+
+query_interval:
+    Time unit how often the csp-billng-adapter collectd the data from the application.
```

Comment:
s/collectd/collects/ 

Also should specify the units we expect, i.e. "Time in seconds"

Reply:
fixed in https://github.com/SUSE-Enceladus/csp-billing-adapter/pull/38